packages/testing/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@netlify/testing",
+  "private": true,
+  "type": "module",
+  "exports": "./lib/index.js",
+  "main": "./lib/index.js",
+  "types": "./lib/index.d.ts",
+  "files": [
+    "lib/**"
+  ],
+  "engines": {
+    "node": "^12.20.0 || ^14.14.0 || >=16.0.0"
```

Comment:
Since we dropped node v12 in the CLI project do we need to drop it here as well?

Reply:
Something for a different PR but yea that's my plan